ufo2fontir/src/source.rs

```diff
@@ -639,29 +640,31 @@ impl Work<Context, WorkError> for StaticMetadataWork {
                 })
             })
             .unwrap_or_default()
-            | font_info_at_default
+            // Also set any bits implied by the style map style name
+            | match font_info_at_default
                 .style_map_style_name
```

Comment:
you're now defaulting to StyleMapStyle::Regular, but you still aren't checking for openTypePreferredSubfamilyName as a fallback when missing styleMapStyleName, see my earlier comment:

> in the case of styleMapStyleName it has a dynamic fallback that checks for openTypeNamePreferredSubfamilyName (which in turn defaults to fontinfo's styleName if missing) and checks if the latter is in the RIBBI (Regular, Italic, Bold, Bold Italic) set of names otherwise defaults to "regular".

you may argue "garbage in, garbage out" but if we want to match 100% ufo2ft, then we need to implement similar fallback logic

Reply:
I agree we should match but I want to land this chunk first so I will add ^ to #300